packages/react-server/core/ClientController.js

```diff
@@ -55,6 +55,7 @@ class ClientController extends EventEmitter {
 
 		window.__reactServerTimingStart = window.performance ? window.performance.timing.navigationStart : new Date;
 		var wakeTime = new Date - window.__reactServerTimingStart;
+		var aboveTheFold = window.performance ? (window.__displayAboveTheFold - window.__reactServerTimingStart) : 0;
```

Comment:
So... this avoids _negative_ numbers, but it still reports _incorrect_ values.

Seems like if we don't have `window.performance` we don't have anything useful to report, so we just shouldn't report at all?  That skews results because some data points are omitted, but it may skew _by less_ than reporting _incorrect_ data points would.